package.json

```diff
@@ -22,7 +23,8 @@
   "devDependencies": {
     "eslint": "4.x",
     "eslint-plugin-prettier": "2.x",
-    "prettier": "1.x"
+    "prettier": "1.x",
+    "tap": "^12.0.1"
```

Comment:
Can you please remove the `tap` dependency, the `test` script, the test file(s), `.eslintignore`, etc? I'd like this PR to only have the minimal code changes, let's setup a test environment later for this. I don't want to be married to this tap package at this point. Thanks!